pkg/cmd/workspace/code.go

```diff
@@ -204,3 +204,78 @@ func init() {
 	ideListStr := strings.Join(ids, ", ")
 	CodeCmd.Flags().StringVarP(&ideFlag, "ide", "i", "", fmt.Sprintf("Specify the IDE (%s)", ideListStr))
 }
+
+func StartIDE(ideId string, activeProfile config.Profile, workspaceId string, workspaceName string, projectName string, providerMetadata string, verbose bool) error {
+	if ideId == "" {
```

Comment:
Because `ideID` is a "required argument" I suggest you remove this and always pass the IDE from `c.DefaultIdeId` when calling the function

Reply:
done